ui/models.py

```diff
@@ -219,7 +221,11 @@ def subtitle_key(self, language='en'):
         Returns:
             str: S3 object key
         """
-        return 'subtitles/{}/subtitle_{}.vtt'.format(self.hexkey, language)
+        return 'subtitles/{key}/subtitles_{key}_{dt}_{lang}.vtt'.format(
+            key=self.hexkey,
+            dt=datetime.now(tz=pytz.UTC).strftime('%Y%m%d%H%M%S'),
```

Comment:
If the particular format doesn't matter here, can we use `datetime.isoformat()` instead? [ISO-8601](https://xkcd.com/1179/) should also have the properly where it can be sorted and it is ordered by time

Reply:
That format doesn't play nice with S3/Cloudfront (it will upload fine, but then can't be downloaded or displayed in video) - my guess is that the colon is the issue.  I'd prefer to keep the date part purely numeric, and include the video hex key too because it might be somewhat useful in figuring out which video it belongs to after downloading.